app/services/moderator/merge_user.rb

```diff
@@ -2,16 +2,16 @@ module Moderator
   class MergeUser < ManageActivityAndRoles
     attr_reader :keep_user, :admin, :delete_user_id
 
+    def self.call(admin:, keep_user:, delete_user_id:)
```

Comment:
Since I was here already I made this conform more to our other service objects:

1. Main API method called `call`.
2. Class methods before instance methods (a convention often found in Ruby projects).